providers/org/biorxiv/normalizer.py

```diff
@@ -1,22 +1,51 @@
-from share.normalize import *  # noqa
+from share.normalize import links
+from share.normalize.parsers import Parser
+from share.normalize.utils import format_doi_as_url
+
+ctx = links.Context()
+
+
+class Publisher(Parser):
+    name = ctx
+
+
+class Association(Parser):
+    pass
+
+
+class Link(Parser):
+    url = links.RunPython('format_doi', ctx)
+    # identifier will always be DOI
+    type = 'doi'
```

Comment:
For this provider it identifier is only used for DOIs. Is always setting `type` to 'doi' adequate?